packages/python/BUILD

```diff
@@ -6,7 +6,7 @@
 
 
 def build(ctx):
-    srcdir = ctx.extract('Python-3.6.0a2')
+    srcdir = ctx.extract('Python-3.6.0a3')
```

Comment:
Well, that's actually on purpose: the version number eventually needs to be exported to a format that all package managers support, having the same total ordering. That's pretty hard to achieve, so that's why I sometimes use artificial version numbers (e.g., 1.2rc1 -> 1.1.91).